tests/Unit/Scraper/Listeners/ConfigureScraperTest.php

```diff
@@ -112,18 +115,15 @@ public function whenIsCalculatedItShouldReturnExtractedDataAndStoreTheNewConfig(
             Log::getFacadeRoot()
         );
 
-        $this->expectsEvents(Scraped::class);
         $configureScraper->handle(new InvalidConfiguration(new ScrapeRequest($this->url, $this->type)));
+        Event::assertDispatched(Scraped::class);
 
         /** @var Scraped $event */
-        $event = collect($this->firedEvents)->filter(function ($event): bool {
-            $class = Scraped::class;
-
-            return $event instanceof $class;
-        })->first();
-        self::assertEquals(
+        $firedEvents = collect(Event::dispatched(Scraped::class));
```

Comment:
Why collect is needed here or in the other tests?

Reply:
It is not required because we have no need to return the array of all event's instances. Removed it.